src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -306,12 +313,12 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
     // Note that components are responsible for doing the right thing based on whether they're
     // active or not.
     function onUrlChanged() {
-        setActiveTab();
-        modeOptionsControl.setMode(UserPreferences.getPreference('mode'));
-        directionsControl.setFromUserPreferences();
-        exploreControl.setFromUserPreferences();
         directionsFormControl.setFromUserPreferences();
+        directionsControl.setFromUserPreferences();
+        modeOptionsControl.setMode(UserPreferences.getPreference('mode'));
         showHideNeedWheelsBanner();
+        setActiveTab();
+        exploreControl.setFromUserPreferences();
```

Comment:
What was malfunctioning with the old sequence?  Whatever works works, but the old way made more conceptual sense to me--setting the tab first so that things that are affected by that know how to behave, then the control components which possibly change things in ways that will inform the Directions and Explore controllers, then the controllers themselves (in either order, since they should be independent).